app/src/main/java/org/literacyapp/battery/BatteryReceiver.java

```diff
@@ -0,0 +1,30 @@
+package org.literacyapp.battery;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.BatteryManager;
+import android.util.Log;
+import android.widget.Toast;
+
+
+public class BatteryReceiver extends BroadcastReceiver {
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+
+        int level = intent.getIntExtra(BatteryManager.EXTRA_LEVEL, -1);
+        int scale = intent.getIntExtra(BatteryManager.EXTRA_SCALE, -1);
+        float batteryPct = level / (float)scale;
```

Comment:
@sladomic I tried with (0), but on the battery status is returning "NaN" and this cannot be use to calculate the percentage.
Battery level:: 0
Battery scale:: 0
Battery status:: NaN

With -1 value.

Battery level:: -1
Battery scale:: -1
Battery status:: 1.0